src/fragments/lib/datastore/flutter/sync/50-selectiveSync.mdx

```diff
@@ -123,4 +123,51 @@ Future<void> initialize() {
 
 `DataStore.configure()` should only by called once.
 
-</Callout>
\ No newline at end of file
+</Callout>
+
+### Advanced use case - Query instead of Scan
+You can configure selective sync to retrieve items from DynamoDB with a [query operation](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.html) against a [Global Secondary Index (GSI)](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html) during the initial or delta sync. By default, the base sync will perform a [scan](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Scan.html). Query operations enable a highly efficient and cost-effective data retrieval for customers running DynamoDB at scale. Learn about creating GSIs with the `@index` directive [here](https://docs.amplify.aws/cli/graphql/data-modeling).
```

Comment:
I feel like the use case here is not clear on why it would be valuable right off the bat. Can we maybe elevate up the last couple of sentences in the paragraph, and elaborate further on the benefit.